packages/calypso-e2e/src/lib/flows/new-post-flow.ts

```diff
@@ -15,6 +18,28 @@ export class NewPostFlow {
 	 */
 	constructor( page: Page ) {
 		this.page = page;
+		this.isLoggedIn = false;
+	}
+
+	/**
+	 * Starts a new post by directly going to the default site of the given
+	 * account. Logs in if necessary.
+	 *
+	 * @param user The username of the account to start the post with
+	 * @returns An instance of the GutenbergEditorPage class
+	 */
+	async startImmediately( user: string ): Promise< GutenbergEditorPage > {
+		const calypsoURL = DataHelper.getCalypsoURL();
+		const siteHost = DataHelper.getAccountSiteURL( user, { protocol: false } );
+
+		await this.page.goto( path.join( calypsoURL, 'post', siteHost ) );
```

Comment:
Wouldn't the following call do the same?

```
await this.page.goto( DataHelper.getCalypsoURL( `post/${siteHost}`)  )
```

The `DataHelper.getCalypsoURL` call already formats the whole URL scheme.

Reply:
https://github.com/Automattic/wp-calypso/pull/58516/commits/f28e039b042538335abb1131a84c7bc2b49630a3